hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -317,6 +317,9 @@ int SaraNcpClient::initParser(Stream* stream) {
         // self->checkRegistrationState();
         // Cellular Global Identity (partial)
         if (r >= 3) {
+            if(r >= 4 && val[3] == 0xFFFFFFFF) {
+                val[3] = 7; // Force SARA R510 fixed value
```

Comment:
Is there a handy meaningful constant we can use instead of `7` ?

Reply:
Unfortunately this AcT for CEREG doesn't line up with the Enums we have for AcT/RaT for u-blox.  We have enums that work with COPS, but they are slightly different for C*REG.  Could update this comment to `Cat-M1` instead of `fixed value`.

Reply:
Updated my comment